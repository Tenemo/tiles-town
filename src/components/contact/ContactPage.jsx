import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'; // eslint-disable-line import/named

const ContactPage = () => {
    return (
        <section className="contactPage">
            <div className="row">
                <div className="col-sm-12 my-3 text-center">
                    <h5>
                        <a href="https://www.linkedin.com/in/ppiech/">
                            <FontAwesomeIcon icon={['fab', 'linkedin']} /> /ppiech
                        </a>
                    </h5>
                    <h5>
                        <a href="https://github.com/Tenemo">
                            <FontAwesomeIcon icon={['fab', 'github']} /> /Tenemo
                        </a>
                    </h5>
                    <h5>
                        <a href="mailto:piotr@piech.dev">
                            <FontAwesomeIcon icon={['far', 'envelope']} /> piotr@piech.dev
                        </a>
                    </h5>
                </div>
            </div>
        </section>
    );
};

export default ContactPage;
