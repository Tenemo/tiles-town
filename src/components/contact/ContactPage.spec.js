import React from 'react';
import ContactPage from './ContactPage';

describe('ContactPage', () => {
    it('matches snapshot', () => {
        const wrapper = shallow(
            <ContactPage />
        );
        expect(wrapper).toMatchSnapshot();
    });
});