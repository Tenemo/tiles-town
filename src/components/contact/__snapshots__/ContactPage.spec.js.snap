// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactPage matches snapshot 1`] = `
<section
  className="contactPage"
>
  <div
    className="row"
  >
    <div
      className="col-sm-12 my-3 text-center"
    >
      <h5>
        <a
          href="https://www.linkedin.com/in/ppiech/"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Array [
                "fab",
                "linkedin",
              ]
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            transform={null}
          />
           /ppiech
        </a>
      </h5>
      <h5>
        <a
          href="https://github.com/Tenemo"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Array [
                "fab",
                "github",
              ]
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            transform={null}
          />
           /Tenemo
        </a>
      </h5>
      <h5>
        <a
          href="mailto:piotr@piech.dev"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Array [
                "far",
                "envelope",
              ]
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            transform={null}
          />
           piotr@piech.dev
        </a>
      </h5>
    </div>
  </div>
</section>
`;
